packages/firebase_auth/example/lib/register_page.dart

```diff
@@ -84,13 +84,13 @@ class RegisterPageState extends State<RegisterPage> {
       email: _emailController.text,
       password: _passwordController.text,
     );
-    if (user != null) {
-      setState(() {
-        _success = true;
-        _userEmail = user.email;
-      });
-    } else {
-      _success = false;
+    try {
+	setState(() {
+		_success = true;
+		_userEmail = user.email;
+	});
+    } on Exception catch(e) {
```

Comment:
It doesn't seem likely that the try block will throw any exceptions. I think you want to wrap the code before it.